### DISCUSSION TITLE
Scope of W3C web, digital accessibility work

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/wai-mission#4] [High-level]
**What technologies are in scope?**

W3C work includes [Web of Things](https://www.w3.org/ecosystems/web-of-things/), [e-commerce](https://www.w3.org/ecosystems/e-commerce/) and [more](https://www.w3.org/ecosystems/).

AI and emerging technologies bring numerous accessibility issues, e.g., [AI 2023 Report](https://www.w3.org/WAI/research/ai2023/) ([AI and Accessibility discussion](https://github.com/w3c/wai-mission/discussions/25))

Mobile apps are more prevalent than "traditional" browsers and content in many part of the world. ([Mobile app accessibility discussion](https://github.com/w3c/wai-mission/discussions/7))

Web Content Accessibility Guidelines 2 has fairly limited scope. [WCAG2ICT](https://www.w3.org/WAI/standards-guidelines/wcag/non-web-ict/) addresses applying WCAG to software and non-web documents. [W3C Accessibility Guidelines 3](https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/) is exploring a broader scope, as introduced in the [Abstract](https://www.w3.org/TR/wcag-3.0/#abstract).
And, there are many digital accessibility issues beyond that defined scope. And likely where WCAG itself is not a good fit.

In which areas is W3C uniquely positioned to lead accessibility work?

With limited resources, do we keep to a narrowly defined scope? Or, do we work on getting more resources to better address a broader scope?

<hr>

_Related discussions:_
* _[AI and Accessibility discussion](https://github.com/w3c/wai-mission/discussions/25)_
* _[Mobile app accessibility](https://github.com/w3c/wai-mission/discussions/7)_
* _[Low resource regions / Global South accessibility](https://github.com/w3c/wai-mission/discussions/17)_

**Comment by mitchellevan** [comment]
> In which areas is W3C uniquely positioned to lead accessibility work?

One context of this question is: how far beyond web-in-a-browser do we go? For comparison, European standards bodies have curated and written accessibility criteria that go beyond web. It's fine for W3C to choose not to tackle some of these topics, such as the entirety of hardware or perhaps certain subtopics within hardware. The result of declining will be to defer to European and other regional standards. Results of taking on any given topics include ensuring more worldwide perspectives.

**Comment by nigelmegitt** [comment]
Media accessibility (e.g. for videos) is handled through MAUR which is a WAI activity I believe, but the technical standards that support delivery of accessible media experiences all seem to be delivered elsewhere, e.g. IMSC/TTML, DAPT and WebVTT in my group, TTWG, and APIs via those who look after HTML.

Outside W3C there are lots of other groups that do related work too, but the level of understanding of accessibility of interfaces amongst e.g. television apps seems like an opportunity for improvement. I think a lot of apps on TVs, and TVs in general, don't support assistive tech very well, for example.

**Comment by mitchellevan** [comment]
In whatever we choose to do, we have the additional choice of how deeply and thoroughly to do it. For example, if (hypothetically) tiny screens and wearables were given a lower priority, something like a Working Group Note could be a way to provide relevant guidance with less investment than a full standard would require.

**Comment by GeorgeKerscher** [comment]
Other groups are doing good accessibility work that is related to W3C activities. When those groups tackle accessibility issues, the W3C may not need to apply resources, but the work could be amplified by the WAI. For example, the DAISY Consortium is conducting testing of Reading Systems that uses the W3C EPUB 3.3 Standard. We could use more people helping us out, and getting recognition from the W3C could help. I know this is contraversial, because how does the W3C know if the various groups are really good at what they do. Never the less, thiese groups doing good work should be praised and refered to by the W3C.
